includes/components/class-form.php

```diff
@@ -581,6 +581,9 @@ public function enqueue_scripts() {
 				wp_script_add_data( 'recaptcha', 'async', true );
 				wp_script_add_data( 'recaptcha', 'defer', true );
 			}
+
+			// Enqueue password strength meter.
+			wp_enqueue_script( 'password-strength-meter' );
```

Comment:
If the only condition is front-end, add this as a dependency for hivepress-frontend, using our script config is always more preferred since it allows filtering.